contrib/coccinelle/zero_trace_reason.cocci

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-2.0
+/// Prevent passing 0 as a reason to send_trace_notify(), because 0 is a valid
+/// value corresponding to new conntrack connections, and passing it instead of
+/// TRACE_REASON_UNKNOWN may confuse Hubble.
+// Confidence: Low
```

Comment:
You can increase this to Medium at least: the risk of false positives is very low.